web/tests/functional/diff_remote/test_diff_remote.py

```diff
@@ -35,6 +36,11 @@ def get_severity_level(name):
     return Severity._NAMES_TO_VALUES[name]
 
 
+def str_to_date(date_str):
+    """ Converts the given string to date. """
+    return datetime.strptime(date_str, '%Y-%m-%d %H:%M:%S.%f')
```

Comment:
I think it would be nice to mention what will be returned if the given string can not be converted to datetime?